DartsArmaMods/MineDetectorFixes
# General - Update links, don't break inheritance for items

**When merged this pull request will:**
- Update GitHub links to point to new repository
- Update HEMTT setup to latest version
  - Moving launch options to `launch.toml`
- Optimize GitHub actions
- Add `authors` property to CfgPatches
- Update mod to no longer break inheritance
  - Requires https://github.com/acemod/ACE3/pull/10366 to work with ACE Arsenal
  - Old versions of cTab will have inheritance broken, but people should move away from old versions anyway.

### Important
- [x] If the contribution adds a new compatibility patch, make sure to update the `docs/README.md` file.
- [x] [Development Guidelines](../.github/CONTRIBUTING.md) are read, understood and applied.
- [x] Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.


### Known Issues
- None
